canonical/ubuntu.com
# WD-12827 - rebrand /azure/private-offer

## Done

Rebrand /azure/private-offer. No content was changed, layout only

## QA

- [demo link](https://ubuntu-com-14156.demos.haus/azure/private-offer)
- [figma](https://www.figma.com/design/MinLlXouBnEDqaoAL3bTWC/24.10-Azure-bubble?node-id=1-6057&t=gyQr0wjIOwcXzRPS-0)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to figma

## Issue / Card
[WD-12827](https://warthogs.atlassian.net/browse/WD-12827)

[WD-12827]: https://warthogs.atlassian.net/browse/WD-12827?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
